src/bacnet/basic/object/msv.c

```diff
@@ -38,6 +39,8 @@
 #include "bacnet/basic/object/msv.h"
 #include "bacnet/basic/services.h"
 
+#define PRINTF printf
```

Comment:
if it's just a capitalisation of the same, why not just leave it as `printf` then?